tendermint/tests/integration.rs

```diff
@@ -35,7 +35,6 @@ mod rpc {
     async fn abci_info() {
         let abci_info = localhost_rpc_client().abci_info().await.unwrap();
 
-        assert_eq!(&abci_info.version, "0.17.0");
```

Comment:
See the specific commit message for my reasoning here: b452fcc6d807fe616bcd16ccc99839649d86f212

An important caveat (tho not well placed in the git logs), is that I don't have a very informed position, given that this is my first tiny foray into helping here, so please forgive if my suggestions are wide of the mark.

Some alternatives that might make sense, in addition to the approach I propose in the commit message:

- Specify the expected version via an environment variable during testing (allowing us to support testing against multiple versions, as desired).
-  Change this to a check for a string that matches a semantic version (e.g., `\d+\.\d+\.\d+`)